dwave/cloud/cli.py

```diff
@@ -584,6 +584,7 @@ def sample(*, config_file, profile, endpoint, region, client_type, solver_def,
         endpoint=endpoint, region=region,
         client=client_type, solver=solver_def)
 
+    t0 = timer()
```

Comment:
Consider making a timer dict so that instead of t0, t1 etc today and then possibly t0.5 tomorrow, you can have `{'start': ?, 'fetch': ? ...}`